app/views/repairs/index.html.erb

```diff
@@ -22,8 +22,8 @@
       </div>
       <div class="row no-gutters">
         <div class="col-1"></div>
-        <div class="col-1"><%= button_to "Edit Repair", edit_repair_path(repair), class: "btn btn-primary btn-sm" %></div>
-        <div class="col-1"><%= button_to "Delete Repair", repair_path(repair), method: :delete, class: "btn btn-danger btn-sm" %></div>
```

Comment:
Actually, it was working to do button_to for Delete Repair.  Does the delete button still work as expected when it is a link_to?   For the demo, we only changed button_to to link_to for Edit Repair, and left the other button as is.